packages/tasks/src/model-libraries.ts

```diff
@@ -314,6 +314,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path:"liveportrait/landmark.onnx"`,
 	},
+	"llama-cpp-python": {
+		prettyLabel: "llama-cpp-python",
+		repoName: "llama-cpp-python",
+		repoUrl: "https://github.com/abetlen/llama-cpp-python",
+		snippets: snippets.llama_cpp_python,
+		filter: true,
```

Comment:
```suggestion
```

Reply:
models won't be tagged explicitly as llama-cpp-python through https://github.com/huggingface-internal/moon-landing/pull/10885, only implicitly

Reply:
let's remove it ok?